sync/sync-api/src/main/java/com/duckduckgo/sync/api/engine/FeatureSyncStore.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2023 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.sync.api.engine
+
+interface FeatureSyncStore {
```

Comment:
I'm unsure what this interface for since we have `SyncStateRepository`? is this a leftover?